plugins/sdm/lib/test_util.py

```diff
@@ -36,6 +36,37 @@ def test_dont_hide_resource_when_tag_doesnt_exist(self):
         sdm_resource.tags = {'hide-resource': 'true'}
         assert is_hidden_resource(config, sdm_resource) is False
 
+class Test_is_hidden_role:
```

Comment:
Let's reuse the method for `is_hide_resource`. It's exactly the same implementation, but for roles.